static/js/browser/browser_page.tsx

```diff
@@ -113,6 +114,7 @@ export class BrowserPage extends React.Component<
             </h2>
           </>
         )}
+        <GoogleMap dcid={this.props.dcid}></GoogleMap>
```

Comment:
this needs a bit more top-padding or margin (should match what's below it on the page)

Reply:
Added more padding via CSS